templates/documents/general_view.html

```diff
@@ -109,16 +109,24 @@
 <table valign="top" width="100%">
     <tr>
         <td>
+            <div style="margin-bottom: 6px;padding-bottom: 6px;border-bottom:3px solid gray;">
+            <a class="css_button">{$file->get_url_web()}</a>
+            <label for='chkShow' class="text" style="margin-left:20px;"><b>{xl t='View contents'}</b>
```

Comment:
ok, I used it again. Showing the document by default with nothing else is much more intuitive for physicians. I recommend the following to really clean this up:
1. Change toggle label to "View Details"
2. Untoggle it by default

So, then the behavior will be that it only shows the document when click on it. And then if the user wants to see all the other information, then can click on "View Details"